resources/views/dash.blade.php

```diff
@@ -1,7 +1,10 @@
+
 @extends('layouts.app')
 
 @section('content')
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```

Comment:
I don't *think* we need this defined here. This blade uses the app.blade.php file located in resources/views/layouts/app.blade.php and JQuery is already included in a script tag in the <head> of that layout.